themes/classic/templates/_partials/header.tpl

```diff
@@ -21,8 +21,11 @@
             <img class="logo img-responsive" src="{$shop.logo}" alt="{$shop.name}">
           </a>
         </div>
-        <div class="col-md-10">
-          {hook h='displayTop'}
+        <div class="col-md-10 position-static">
+          <div class="row">
+            {hook h='displayTop'}
+            <div class="clearfix"></div>
```

Comment:
I think the result might be slightly different, but not visible.
http://stackoverflow.com/questions/8554043/what-is-clearfix

Anyway it's not very important :D